crates/distribution-types/src/lib.rs

```diff
@@ -467,7 +485,7 @@ impl SourceDist {
         }
     }
 
-    /// Returns the path to the source distribution, if if it's a local distribution.
+    /// Returns the path to the source distribution, if it's a local distribution.
     pub fn as_path(&self) -> Option<&Path> {
         match self {
             Self::Path(dist) => Some(&dist.path),
```

Comment:
I think all the changes in this file could've been done as a tiny standalone PR, to reduce the size of the diff.

Reply:
Good idea: #3320